src/commands/protocols/publish/receiver/calculate-proofs-command.js

```diff
@@ -55,23 +76,34 @@ class CalculateProofsCommand extends EpochCommand {
                 hashFunctionId,
                 agreementData,
                 operationId,
+                assertionId,
```

Comment:
Should we add `stateIndex` here as well?

Reply:
we are calculating stateIndex in command, we don't need to pass it